service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/CommandManager.java

```diff
@@ -107,28 +109,51 @@ private void handleError(Exception e, CommandRequest requestBO) {
     @SuppressWarnings("PMD")
     public void configPush(CommandRequest request) {
```

Comment:
This method has become quite long, would it be possible to break the content into smaller method calls?